### PR TITLE
BAU: Migrate JwtVerificationHandler to use slf4j to standardise on a single logging library. 

### PR DESCRIPTION
**Pull request by Danomanic**
### What changed

Migrate JwtVerificationHandler to use slf4j to standardise on a single logging library. 

Also upgraded gradle packages and standardised on quotations in gradle build files.

### CODE REVIEW

**Review by dannyhayes-gds — APPROVED** [review]

**Review by danworth — DISMISSED** [review]
Looks good, thanks for standardising the loggers 👍  The package bumps are all patches besides nimbus which jumps a few minor version but since CI passes I think we can assume its all compatible 🏅

**Comment by Danomanic** [comment]
Just noticed #61 is open which makes significant changes to the `JwtVerificationHandler`. I will wait until that is merged before merging this one.

**Review by danworth — APPROVED** [review]
Looks good 👍

**Pull request merged**
